website/docs/r/healthcare_dicom.html.markdown

```diff
@@ -58,6 +66,30 @@ An `identity` block supports the following:
 
 * `identity_ids` - (Optional) A list of User Assigned Identity IDs which should be assigned to this Healthcare DICOM service.
 
+---
+
+A `cors` block supports the following:
+
+* `allowed_origins` - (Optional) A list of allowed origins for CORS.
+
+* `allowed_headers` - (Optional) A list of allowed headers for CORS.
+
+* `allowed_methods` - (Optional) A list of allowed methods for CORS.
+
+* `max_age_in_seconds` - (Optional) The maximum age in seconds for the CORS configuration (must be between 0 and 99998 inclusive).
+
+* `allow_credentials` - (Optional) Whether to allow credentials in CORS. Defaults to `false`.
+
+---
+
+A `storage` block supports the following:
+
+* `file_system_name` - (Required) The filesystem name of connected storage account. Changing this forces a new Healthcare DICOM Service to be created.
+
+* `storage_account_id` - (Required) The resource id of connected storage account. Changing this forces a new Healthcare DICOM Service to be created.
+
+~> **Note:** The `is_hns_enabled` needs to be set to `true` for the storage account to be used with the Healthcare DICOM Service.
```

Comment:
If this is the case then shouldn't this be set in the `complete` test config?

Reply:
This property is currently set for `azurerm_storage_account` resource in the complete test